lua/telescope/make_entry.lua

```diff
@@ -344,6 +344,7 @@ function make_entry.gen_from_lsp_symbols(opts)
 
   local displayer = entry_display.create {
     separator = " ",
+    hl_chars = { ['['] = 'TelescopeBorder', [']'] = 'TelescopeBorder' },
```

Comment:
We can do this but this is not in master right now. So it will currently not apply the right color. But if this is merged i will rebase my PR immediately so everything comes together.